### ISSUE TITLE
Raise bootstrap version to latest release

### ISSUE THREAD

**Issue by arndmesser** [misterboe/startpilot#75]
"bootstrap": "^4.0.0", -> "bootstrap": "^4.1.3",

**Comment by DanielRuf** [comment]
> ^

It's basically the same with this SemVer selector as it upgrades packages. Just the lockfiles are problematic when it comes to npm. `npm update <package>` is usually the needed way (unfortunately also `npm i` updates the lockfile too).

**Comment by DanielRuf** [comment]
See https://docs.npmjs.com/misc/semver#caret-ranges-123-025-004

**Comment by arndmesser** [comment]
Yes indeed.
But as soon as you have installed startpilot, it's not concatenated with this repo anymore. There are no restrictions for maintaining your project. 

For instance we change the patchlevel to '*' as soon as a project is in a final stage. 
This is a philosophy which differs from one agency to another.

The lock files are the way we handle "package versions" in a team and for our deployment. As soon it is there, everybody is on the same version, just with `npm install`. There ist no update until someone does it, for whatever reason.

... anyway!
Which way do you prefer for this project?

**Comment by DanielRuf** [comment]
In general I would recommend using `npm ci` (for npm 6+) or `yarn` + the `~` SemVer selector (same as 1.0.*).

Well, actually `npm install` automatically updates packages (at least with npm 6).

The new lockfiles from npm 5 are different than `npm shrinkwrap`, especially when it comes to the `npm i` command =)

**Comment by DanielRuf** [comment]
> just with npm install. There ist no update until someone does it, for whatever reason.

Your setup.sh does it.

**Comment by DanielRuf** [comment]
> just with npm install. There ist no update until someone does it, for whatever reason.

Your start.sh does it.

**Comment by DanielRuf** [comment]
> gulp build

I guess this is not mentioned. In the readme gulp is not mentioned, that `npm i -g gulp gulp-cli` has to be done before the start.sh script would work without any issues.

Shall I create a new issue regarding this and the other things?

**Comment by martin-eberle** [comment]
Hi @DanielRuf, yes indeed, the step to install gulp and gulp-cli globally is missing. Would you like to create a new issue or pull request regarding this? Thanks a lot.

**Comment by DanielRuf** [comment]
> Hi @DanielRuf, yes indeed, the step to install gulp and gulp-cli globally is missing. Would you like to create a new issue or pull request regarding this? Thanks a lot.

There you go: https://github.com/Startpiloten/startpilot/pull/77

I also made it a bit more clear that these are actual system requirements.
